### PR TITLE
[Feature #84852] [SPRINT] Allow creation of ProjectPlanningTime from ProjectTask

### PR DESCRIPTION
**Pull request by psh-axelor**

### CODE REVIEW

File: axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml [hunk 1]
```diff
@@ -264,6 +273,21 @@
...
+    <attribute name="readonly" for="projectTask" expr="eval: true"/>
+    <attribute name="value" for="timeUnit,displayTimeUnit"
+      expr="eval: __repo__(ProjectTask).find(_projectTask?.id)?.timeUnit"/>
+    <attribute name="value" for="project" expr="eval: _projectTask?.project"/>
+    <attribute name="value" for="employee" expr="eval: __user__?.employee"
+      if="__user__ in __repo__(Project).find(_projectTask?.project?.id)?.membersUserSet"/>
+    <attribute name="value" for="product" expr="eval: __user__?.employee?.product"
+      if="__user__ in __repo__(Project).find(_projectTask?.project?.id)?.membersUserSet &amp;&amp; __user__?.employee"/>
+    <attribute name="value" for="startDateTime" expr="eval: _sprintFromDate"/>
+    <attribute name="value" for="endDateTime" expr="eval: _sprintToDate"/>
```
Comment (rth-axelor) [inline axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml:288]: Remove the last 2 attributes and create a new function that take the startDateTime and endDateTime from the projectTask's sprint and use this function at the end of the onNew and the projectTask's onChange

File: axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml [hunk 2]
```diff
@@ -148,8 +148,15 @@
...
-    <context name="_project" expr="eval:__parent__"/>
+    <context name="_project" expr="eval:__parent__"
+      if="__parent__?._model == 'com.axelor.apps.project.db.Project'"/>
+    <context name="_projectTask" expr="eval:__parent__"
+      if="__parent__?._model == 'com.axelor.apps.project.db.ProjectTask'"/>
     <context name="_timePlannedPopup" expr="eval: true"/>
+    <context name="_sprintFromDate"
+      expr="eval: _projectTask?.sprint?.sprintPeriod?.fromDate?.atStartOfDay()"/>
+    <context name="_sprintToDate"
+      expr="eval: _projectTask?.sprint?.sprintPeriod?.toDate?.atStartOfDay()"/>
```
Comment (rth-axelor) [inline axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml:159]: No need of _sprintFromDate and _sprintToDate

File: axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml [hunk 3]
```diff
@@ -148,8 +148,15 @@
     <view-param name="show-toolbar" value="false"/>
     <view-param name="popup-save" value="false"/>
     <view-param name="forceTitle" value="true"/>
-    <context name="_project" expr="eval:__parent__"/>
+    <context name="_project" expr="eval:__parent__"
+      if="__parent__?._model == 'com.axelor.apps.project.db.Project'"/>
+    <context name="_projectTask" expr="eval:__parent__"
+      if="__parent__?._model == 'com.axelor.apps.project.db.ProjectTask'"/>
```
Comment (rth-axelor) [inline axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml:154]: Add a 
<context name="_project" expr="eval:__parent__.project"
      if="__parent__?._model == 'com.axelor.apps.project.db.ProjectTask'"/>
Please

**Comment by psh-axelor** [comment]
will resolve conversation in https://github.com/axelor/axelor-open-suite/pull/13415

**Pull request closed without merging**